docs/getting/tutorial.rst

```diff
@@ -0,0 +1,88 @@
+.. _tutorial:
+
+**************************
+Tutorial
+**************************
+
+This example will show the simplist way to use pandas with pint and the underlying objects.
+It's slightly fiddly to set up units compared to reading data and units from a file.
+A more typical use case is given in :doc:`Reading from csv <../user/reading>`.
+
```

Comment:
A key to understanding the behavior of PintArrays within Pandas DataFrames is that PintArrays are ExtensionArrays, and Pandas supports only 1-dimensional ExtensionArrays.  In a 2-dimensional DataFrame, PintArrays are only ever columns.  If you ask for a single row (via `.loc` or `.iloc` or some such), you will get back a Series, that *may* have a PintArray as its `values` (for example when the entire DataFrame is homogeneous in its units).  But that Series is just a view constructed by PintPandas for your convenience.  Rows in the DataFrame itself are not PintArrays.